plugins/janus_streaming.c

```diff
@@ -6328,6 +6359,14 @@ static int janus_streaming_rtsp_parse_sdp(const char *buffer, const char *name,
 	return 0;
 }
 
+static inline gint64 janus_min_if(gint64 a, gint64 b)
+{
+	return a > 0
+		? (a > b ? b : a)
+		: b
+	;
+}
```

Comment:
The function name prefix should be `janus_streaming_`, not just `janus_` (see above).
Please put the opening bracket on the same line as the function name (code style).
The action itself is simple enough to be on a single line, rather than four as you wrote it.
There should be a comment on top that explains what the function is about (e.g., "Helper function to ...")

Reply:
You could probably just implement this with https://developer.gnome.org/glib/stable/glib-Standard-Macros.html#CLAMP:CAPS

Reply:
That macro has a little bit different functionality.